src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -153,18 +156,46 @@ public void onSpoutOpen(
 
         createSidelineTriggers();
 
+        Preconditions.checkArgument(
+            spoutConfig.containsKey(SidelineConfig.REFRESH_INTERVAL_SECONDS)
+            && spoutConfig.get(SidelineConfig.REFRESH_INTERVAL_SECONDS) != null,
+            "Refresh interval is required."
```

Comment:
Probably best to show which config option specifically is missing.

Reply:
Done